scripts/generate/consts.go

```diff
@@ -122,6 +123,35 @@ func generateParseModeConsts() string {
 	return out.String()
 }
 
+func generateChatActionConsts() string {
+	chatActions := []string{
+		"typing",
+		"upload_photo",
+		"record_video",
+		"upload_video",
+		"record_voice",
+		"upload_voice",
+		"upload_document",
+		"choose_sticker",
+		"find_location",
+		"record_video_note",
+		"upload_video_note",
+	}
```

Comment:
Would it be possible for this list to be obtained from the type spec?
For example, by using some regexes on the sendChatAction method's "action" field description? 

If the list is defined manually, then it might as well not be generated; it's just a manually written list, and any changes on telegrams side wouldn't reflect.
I know it's done this way for formatting consts, but that was intended as an exception; parsemodes aren't expected to change, and are relatively short list!